### PR TITLE
Hierarchy fixed, all features share path

### PR DESCRIPTION
**Pull request by ZimGil**
"rootFolder" was moved to the app.component and is now shared with all other components and is part the requests of all app features.

Solves #10

### CODE REVIEW

File: src/app/components/generate/generate.component.ts [hunk 1]
```diff
@@ -10,21 +11,24 @@ import { AngularClassOptions } from '../../models/angular-class-options.interfac
 })
 export class GenerateComponent {
 
+  @Output() sendCommand = new EventEmitter<AngularCliCommand>();
+  command: AngularCliCommand;
   name = 'Name';
```
Comment (ortichon) [inline src/app/components/generate/generate.component.ts]: Why do we need this property initialization? can't we just declare it?

File: src/app/components/serve/serve.component.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
-import { Component, Input } from '@angular/core';
+import { Component, Output, EventEmitter } from '@angular/core';
+import { AngularCliCommand } from '../../models/angular-cli-command.interface';
```
Comment (ortichon) [inline src/app/components/serve/serve.component.ts]: Add empty line before this one

File: src/app/models/angular-cli-command.interface.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+export interface AngularCliCommand {
```
Comment (ortichon) [inline src/app/models/angular-cli-command.interface.ts]: What is this?

**Comment by ZimGil** [comment]
All change requests were addressed to...

Please review again @ortichon

**Pull request closed without merging**